modules/serverless-monitoring-services-custom-metrics.adoc

```diff
@@ -0,0 +1,73 @@
+[id="serverless-monitoring-services-custom-metrics_{context}"]
+= Knative service with custom application metrics
+
+You can extend the set of metrics exported by a Knative service. The exact implementation depends on your application and the language used.
+
+The following listing implements a sample Go "hello world" application that exports the count of processed events custom metric.
```

Comment:
>  implements a sample Go "hello world" 

I assume that is THE application used inside the above image?

Do we want to host the code also on our own repo? So we have it versioned in git, rather than just on the docs?

Reply:
@matzew I'm not sure now, need to verify with @skonto .

Reply:
I have no problem pushing the code somewhere. Where that could be?

Reply:
@skonto Matthias' suggestion of Quay sounds good.

Is it correct that the listing in this document that I'm quoting is the one used in that image?

Reply:
Yes the image there uses that code. I dont have access to the quay repo btw.

Reply:
I have the image under `quay.io/skontopo/helloworld-go:metrics` but cant push it under openshift-knative account.

Reply:
We'll resolve the new image location after the release.

Reply:
https://quay.io/repository/openshift-knative/helloworld-go:metrics its now available